src/main/kotlin/org/rust/ide/intentions/CreateFunctionIntention.kt

```diff
@@ -60,11 +60,11 @@ class CreateFunctionIntention : RsElementBaseIntentionAction<CreateFunctionInten
         val path = element.parentOfType<RsPath>()
         val functionCall = path?.parentOfType<RsCallExpr>()
         if (functionCall != null) {
-            if (!path.isUnresolved) return null
+            if (path.resolveStatus == PathResolveStatus.RESOLVED) return null
```

Comment:
Does this intention make sense in case of a malformed path?
Looks like we should use `!= PathResolveStatus.UNRESOLVED` here